fonts/fonts.css

```diff
@@ -0,0 +1,26 @@
+@font-face { 
+  font-family: 'Montserrat';
+  src: url('/fonts/bold/montserrat-bold-webfont.woff2') format('woff2'),
+    url('/fonts/bold/montserrat-bold-webfont.woff') format('woff');
+  font-display: swap;
+  font-weight: bold;
+  font-style: normal;
+ }
+
+@font-face {
+  font-family: 'Montserrat';
+  src: url('/fonts/regular/montserrat-regular-webfont.woff2') format('woff2'),
+    url('/fonts/regular/montserrat-regular-webfont.woff') format('woff');
+  font-display: swap;
+  font-weight: normal;
+  font-style: normal;
+ }
+
+@font-face {
+  font-family: 'Montserrat';
+  src: url('/fonts/medium/montserrat-medium-webfont.woff2') format('woff2'),
+    url('/fonts/medium/montserrat-medium-webfont.woff') format('woff');
+  font-display: swap;
+  font-weight: medium;
+  font-style: normal; 
+ }
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/GurevAleksey_itogovaya/pullRequest?prid=6088340)